static/js/factories/dashboard.js

```diff
@@ -129,6 +129,16 @@ export const makeCoupon = (program: Program): Coupon => ({
   object_id: program.id,
 });
 
+export const makeCourseCoupon = (courseId: number, programId: number): Coupon => ({
```

Comment:
Can you change `courseId` and `programId` to `course` and `program` for consistency with `makeCoupon`?